stackslib/src/chainstate/stacks/boot/pox-4.clar

```diff
@@ -117,7 +118,9 @@
         ;;  previous stack-stx calls, or prior to an extend)
         reward-set-indexes: (list 12 uint),
         ;; principal of the delegate, if stacker has delegated
-        delegated-to: (optional principal)
+        delegated-to: (optional principal),
+        ;; signing key for Nakamoto, only 'none' when delegated & before delegate calls stack-aggregation-commit-indexed
```

Comment:
I don't think this is ever `none` if we're setting the signing key on `delegate-stack-stx` (which IIRC is the plan now)